src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java

```diff
@@ -1542,6 +1548,42 @@ public boolean configure(StaplerRequest req, JSONObject o) throws FormException
 				breakdownPageSize = DEFAULT_PAGE_SIZE;
 			}
 
+			try {
+				String connectTimeoutString = o.getString("connectTimeout");
```

Comment:
Sorry, I know this is not named by you, but can you please rename o to something else like obj, json or jsonObject? 'o' is a little too short and hard to understand...

Reply:
Renamed